junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvArgumentsProvider.java

```diff
@@ -26,17 +28,30 @@
 import org.junit.jupiter.params.support.AnnotationConsumer;
 import org.junit.platform.commons.PreconditionViolationException;
 import org.junit.platform.commons.util.Preconditions;
+import org.junit.platform.commons.util.ReflectionUtils;
 import org.junit.platform.commons.util.UnrecoverableExceptions;
 
 /**
  * @since 5.0
  */
 class CsvArgumentsProvider implements ArgumentsProvider, AnnotationConsumer<CsvSource> {
 
-	private static final Pattern NEW_LINE_REGEX = Pattern.compile("\\n");
-
 	private static final String LINE_SEPARATOR = "\n";
 
+	private static final Method stripIndentMethod;
+
+	static {
```

Comment:
Not a necessity, but maybe this block and `stripIndent(String textBlock)` could be moved in a different file, for the sake of readability?

Reply:
Good point. I'll go ahead and create a local version of the previously discussed `TextBlockUtils` class and move those there.

Reply:
done ✅